cmd/tracee-ebpf/main.go

```diff
@@ -355,7 +355,7 @@ func prepareBpfObject(config *tracee.Config, kConfig *helpers.KernelConfig, OSIn
 	kVersion = OSInfo.GetOSReleaseFieldValue(helpers.OS_KERNEL_RELEASE)
 	kVersion = strings.ReplaceAll(kVersion, ".", "_")
 
-	bpfFilePath = fmt.Sprintf("/tmp/tracee/tracee.bpf.%s.%s.o", kVersion, tVersion)
+	bpfFilePath = fmt.Sprintf("%s/tracee.bpf.%s.%s.o", traceeInstallPath, kVersion, tVersion)
```

Comment:
Don't we want to support looking for the bpf object in the same directory of the executable like we did in the past? I use it many times when I check on non-core environments (so I don't have to install the bpff object)

Reply:
Why don't you use `sudo TRACEE_EBPF_FILE=./file ./dist/tracee-ebpf ...` instead ? You can point anywhere you want. WDYT ?

Reply:
Because I'm too lazy and don't want to write `TRACEE_BPF_FILE=./file` :-)

Reply:
I'll check that (and see if Makefile.one is ready for that as well).

Reply:
If it's too much of a hassle, we can give it up